src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppRebootWatchdog.java

```diff
@@ -49,13 +53,33 @@ public class SCAppRebootWatchdog {
   /** Status indicator if reboot watchdog is serviced */
   private static boolean isRebootWatchdogServiced = false;
 
+  /** The time the watchdog will accept servicing */
+  private static long watchdogReadyTimeMillis;
+
   /**
-   * Enables the reboot watchdog.
+   * Enables the reboot watchdog. The watchdog will be enabled with a default service time of 0
+   * seconds. This means that when using the default watchdog can be serviced immediately after
+   * initialization. This ensures compatibility with existing applications that have implemented the
+   * reboot watchdog but have not incorporated a sizeable delay between watchdog initialization and
+   * servicing. For new applications this method should not be used and instead the method with a
+   * service time parameter should be used.

```

Comment:
```suggestion
   * Enables the reboot watchdog. 
   * <p>
   * The watchdog will be enabled with a default service time of 0 seconds. This means that when
   * using the default service time, the watchdog can be serviced immediately after initialization. 
   * This ensures compatibility with existing applications that have implemented the reboot
   * watchdog, but have not incorporated a sizeable delay between watchdog initialization and
   * servicing.
   * <p>
   * For new applications, this method should not be used. Instead, the 
   * {@link #enableRebootWatchdog(int)} method should be used.
```

Reply:
Grammar and spacing changes for readability.

Reply:
changed